js/main.js

```diff
@@ -0,0 +1,25 @@
+var $form = document.querySelector('form');
+var $img = document.querySelector('img');
+var $title = document.querySelector('#title');
+var $url = document.querySelector('#url');
+var $notes = document.querySelector('#notes');
+$url.addEventListener('input', function (event) {
+  $img.setAttribute('src', $url.value);
+});
+var counter = 0;
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var entry = {
+    title: $title.value,
+    url: $url.value,
+    notes: $notes.value,
+    nextEntryId: 0
+  };
+  entry.nextEntryId = ++counter;
```

Comment:
`nextEntryId` is a good name for the property in the data model, since its value is the "next" id to use for the "next" entry. This is an id on a single entry though, it doesn't represent the "next" anything. A better name for your property on the `entry` object would be `id` or `entryId`